src/edgepi/pwm/README.md

```diff
@@ -14,16 +14,17 @@ edgepi_pwm = EdgePiPWM()
 
 # Initialize and open pwm device
 edgepi_pwm.init_pwm(PWMPins.PWM1)
-# Setting PWM parameters
-edgepi_pwm.set_config(PWMPins.PWM1, 1000.0, 50.0)
-# enable the pwm signal 
+# Setting PWM parameters, frequency 1K to 10K, duty cycle form 0-1.0
```

Comment:
Should we specify that these are floats?

Like `Frequency in the range 1000.0 - 10000.0, duty cycle in the range 0.0 - 1.0` or something like that.

Reply:
frequency is "int" and the dutycycle is "float"

Reply:
[a4aea98](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/369/commits/a4aea98c56ec92b8ec443c416225e247803b332f)

Reply:
https://github.com/vsergeev/python-periphery/blob/f3afcd7b5a799a066a6cf321e0456a040dd66c2c/periphery/pwm.py#L267

I think I made changes on the SDK that made it a float.